### ISSUE TITLE
whereIn/whereNotIn throwing exception

### ISSUE THREAD

**Issue by davimug** [pdphilip/laravel-elasticsearch#45]
## Package version 
4.3.0

## Describe the bug
Imagine I have an index called `signals` with an `id` property of the type `keyword`. When I run the following:

`Signal::query()->whereIn('id', [5, 10, 15])->get();`

or 

`Signal::query()->whereNotIn('id', [5, 10, 15])->get();`

I get the following error:
`ErrorException Undefined array key "signals"`

Other methods work fine however.

I have downgraded to version:

`4.0.2` 

Which as fixed this issue (this is the version used in production and working fine, now sure about any versions which supersede  this except 4.3.0

**Comment by pdphilip** [comment]
Hey @davimug , please share:

1: Index mapping for your Signal model:
```php
Signal::getIndexMappings();
```

2: Show me the Query Exception:

```php
use PDPhilip\Elasticsearch\DSL\exceptions\QueryException;

try {
    return Signal::query()->whereIn('id', [5, 10, 15])->get();
} catch (QueryException $e) {
    return $e->getDetails(); //Show me this
}

```

Thanks

**Comment by pdphilip** [comment]
Any updates here, @davimug ?

**Comment by pdphilip** [comment]
I haven't heard back from you, so I assume this is no longer an issue. 

I suspect this may have be related to #47, which has a fix in [v4.4.1](https://github.com/pdphilip/laravel-elasticsearch/releases/tag/v4.4.1) so closing this.

Open a new issue with the `Index mapping` and `Query Exception` if there is something else going on
